src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -237,129 +237,119 @@
 
         "fullyQualifiedLogicalName": 
         {
-          "description": "A string containing the language-specific logical name of the location where the result occurs; e.g. C: Foo C++: Namespace::Class::MemberFunction(int, double) const&& C++: Namespace::NonmemberFunction(int, double) C#: SecurityCryptographyRuleTests.DESCannotBeUsed.EncryptData(System.String,System.String,System.Byte[],System.Byte[])",
+          "description": "A string that summarizes the information in logicalLocation in a format consistent with the programming language in which the programmatic constructs expressed by logicalLocation were expressed.",
           "type": "string"
         },
 
         "properties": 
         {
-          "description": "Key/value pairs that provide additional information about this location. This might be used to annotate specific stack frames or points in code with additional information, such as assumed values of variables at that point of execution, etc.",
+          "description": "Key/value pairs that provide additional information about the location.",
           "type": "object",
           "default": {}
         },
 
-        "tags": 
-        {
-          "description": "A unique set of strings that provide additional information for the associated location.",
+        "tags": {
+          "description": "A set of distinct strings that provide additional information about the location.",
           "type": "array",
-          "items":
-          {
+          "items": {
             "type": "string",
-            "default": []
+            "default": [ ]
           }
         }
       }
     },
     "logicalLocationComponent": 
     {
-      "description": "One level (e.g. namespace, function, etc.) of a logical location tree.",
+      "description": "A component of a logical location.",
       "additionalProperties": false,
       "type": "object",
       "properties": {
         "name": 
         {
-          "description": "Name of the item specified by this location component.",
+          "description": "Identifies the construct in which the result occurred. For example, this property might contain the name of a class or a method.",
           "type": "string"
         },
 
         "kind": 
         {
-          "description": "The type of item this location refers to.",
+          "description": "The type of construct this logicalLocationComponent refers to. Should be one of 'method', 'module', 'namespace', 'package', 'resource', or 'type', if any of those accurately describe the construct.",
```

Comment:
Suggest adding binary, assembly, attribute, field, property, struct, class, enum, parameter, local, and lambda to this list. Remove type.  What the heck. We have an issue somewhere to proposes a set of standard values.